### ISSUE TITLE
Custom file upload error

### ISSUE THREAD

**Issue by Maithy15** [tbaccata/amica#11]
Hi Sebastian,

I have a custom output file from DIA-NN. It seems to be getting uploaded successfully but when I press analyze it says no data uploaded. I have attached the files here. Could you please tell me what could be the problem?

Thanks
Maithy

![image](https://user-images.githubusercontent.com/78071837/162220812-5b535a6a-d2a1-4e2e-9fbd-f75b424cc603.png)
[amica_groups.txt](https://github.com/tbaccata/amica/files/8443775/amica_groups.txt)
[amica_specification.txt](https://github.com/tbaccata/amica/files/8443776/amica_specification.txt)
[Pel_Amica_Example.txt](https://github.com/tbaccata/amica/files/8443777/Pel_Amica_Example.txt)
[amica_expt.txt](https://github.com/tbaccata/amica/files/8443778/amica_expt.txt)

**Comment by tbaccata** [comment]
Hi Maithy,

the `...` in the Pattern column of the example specification file is just be a placeholder,
columns that are missing in your data must not be declared in the specification file, which throws an error.

The file Pel_Amica_Example.txt contains 48 Intensity columns, but only 47 are declared in `amica_expt.txt`.
The 2nd sample in Pel_Amica_Example.txt `M08_R1_Pel` is missing in `amica_expt.txt`, amica fails to succesfully assign all samples to groups. 

I've added the sample `M08_R1_Pel` to the group `R1_Pel` at the 2nd position of the file `design.txt` (hope that is correct).
[design.txt](https://github.com/tbaccata/amica/files/8444263/design.txt)

And I've attached you an appropriate `specfication` file.
[specification.txt](https://github.com/tbaccata/amica/files/8444266/specification.txt)

Then it should work.

Best,
Sebastian
